### ISSUE TITLE
Bug: Using italics/bold in item description fails in web creator

### ISSUE THREAD

**Issue by Griatch** [textadventures/quest#814]
The "look at" description editor bugs out when using `<>` markup in text. How to reproduce (Latest Firefox 41.0.1 under Linux): 

1. In web editor, create new object normally
2. Go to "Description" field and choose a "look at" object description->Text and start filling in the large field. 
3. Start with an entry using brackets, like `<i> ... </i>`, for example `<i>Test1</i>`, then save. A comma is seen appended to the end of the description, after the last </i> (this is the first, less critical issue).
4. Now add some other non-italized text after the first, like this (removing the spurious comma that was added: `<i>Test1</i>Test2`. Save. What is now actually appearing is this: `<i>Test1</i>Test2,<i>Test1</i>,` 

No matter how often you try to delete and fix this, the string will just be getting longer and longer. I can only imagine this is some sort of interaction with the look verb under the `Verbs` tab, which shows a string that is not updated but just appended to. Deleting the string here does not help, at all, it actually appends an empty string to the field unde the `Setup` tab. In fact, deleting the look verb completely from the `Verbs` tab seems to be the only way to get out of this.  

This seems to happen for all `<>` formatting markup. All in all, this makes the web editor pretty useless for any object description text with formatting.

**Comment by alexwarren** [comment]
I can't reproduce this. The fact you're seeing a "look" verb on your Verbs tab sets off alarm bells - that shouldn't be there. It sounds like you've added this as a verb somehow, which again is something I can't reproduce.

**Comment by Griatch** [comment]
Alas, I see this extra "look at" verb every time I create an object and starts filling in the look description. 
It's pretty annoying too, since I made a game without textual input - the system creates dual look/look at verbs in the gui and I have to manually turn off "automatic creation of verbs" and/or manually remove the verbs for every single object.

I switched to the desktop version (Win7) and had the same thing happen. That one is luckily not affected by the `<i>...</i>` errors I describe above (which I still get, for every object I create, no matter how bare).
